### ISSUE TITLE
[BUG] chest clothing does not correctly follow arm swing animation

### ISSUE THREAD

**Issue by DoomRater** [Armourers-Workshop/Armourers-Workshop#310]
**Describe the bug**
Punching the air causes the player to animate, but a chestpiece does not seem to follow the player's animation correctly.  Similarly, the arm pieces seem to not quite line up with the arms either.

I retested with only Armourer's loaded, but I tested on 0.49.1.496 and 481 (which was the last dev version I had been using reguarly) and found the behavior didn't quite match.

Some screenshots:
https://imgur.com/a/ilW3Jzb

**Comment by RiskyKen** [comment]
No other mods loaded that edit the player model?

**Comment by DoomRater** [comment]
This is incredibly subtle and I need to capture footage of it to show off what isn't moving correctly.  At first I thought it was fixed when I loaded up 50.1 and started investigating what it looked like with other mods installed, but I can definitely see that the torso isn't quite lining up with a different piece of clothing than the one in the picture.

Color me amazed if Twilight Forest does anything to the player animation model, so I went back and made sure I actually unchecked Twilight Forest this time.  I've updated the imgur post with a video this time, but just in case that doesn't work: https://imgur.com/iNkrWrf
